Lombiq.HelpfulExtensions.Tests.UI/Lombiq.HelpfulExtensions.Tests.UI.csproj

```
@@ -28,6 +28,12 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="OrchardCore.ContentManagement.Abstractions" Version="2.1.7">
```

We keep these intentionally on 2.1.0, see https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions?tab=readme-ov-file#net-and-orchard-core-version-support-policy.

```suggestion
    <PackageReference Include="OrchardCore.ContentManagement.Abstractions" Version="2.1.0">
```